mirdata/groove_midi.py

```diff
@@ -378,7 +380,15 @@ def download(data_home=None):
         data_home, zip_downloads=[AUDIO_MIDI_REMOTE], cleanup=True
     )
 
-    os.rename(os.path.join(data_home, 'groove'), os.path.join(data_home, 'Groove MIDI'))
+    # files get downloaded to a folder called groove - move everything up a level
```

Comment:
thanks. this is a bit slower but indeed the safest option, considering that we can't guarantee that we have write access to `os.path.split(data_home)[0]`